Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<ul><li>"things" repeated twice</li><li>Examples include estimating parameters ... or standard deviation</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Fixed thanks</p>

Reply:
<p>Actually I think we should also have the Exponential in there</p><p> </p>

Reply:
<p>Alex: Why? I don't really see it used in modern Bayesian modeling. I think we really want to be very restrictive in how many we add as to not overwhelm users, and we already have quite a few.</p><p> </p>

Reply:
<p>Not adding exponential :)</p>